gravitee-am-ui/src/app/components/sidenav/sidenav.component.html

```diff
@@ -39,6 +39,29 @@
         <gio-menu-item [icon]="item.icon" [active]="rla.isActive" [iconRight]="item?.iconRight$ | async">{{ item.label }}</gio-menu-item>
       </a>
     </gio-menu-list>
+    <gio-menu-footer *ngIf="showExpirationMessage() && canDisplayOrganizationSettings()">
+      <gio-menu-header>
+        <div
+          style="background-color: white; border-radius: 10px; align-self: flex-end"
```

Comment:
wouldn't it be cleaner to add the all the styles in the css files? Also `align-self: flex-end` may not have any effect due to parent component display is set to block.